NewForce-Cohort-7/tabloidfullstack-bloodyscallwags
# KH view all categories

Description

Trello Ticket #9 View All Categories
<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;QtQYe1ej">#9 View All Categories</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

As an admin I would like to see all the available categories so that I can choose to edit or delete one, or see that I should add a new one.

Given an admin is in the app
When they select the Category Management link in the menu
Then they should be directed to a page that lists all the Category names ordered alphabetically

NOTE: For the time being it is acceptable to treat all users as admin users. There is a future story about enforcing user permissions.
Testing Instructions
git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout <kh-ViewAllCats>
If you don't see the most recent version of my files, git pull origin <Kh-ViewAllCats>
If you haven't created the Tabloid MVC database, please do so.
Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/02_Tabloid_Seed_Data.sql)
Click the execute with debugger solid green button to launch the app
In browser, please go to Login on the nav bar and log in with this email: [foo@bar.com](mailto:foo@bar.com)
This should redirect you back to the home page.
INCLUDE ADDITIONAL STEPS HERE
Checklist:
[] My code follows the style guidelines of this project
[] I have performed a self-review of my code
[] I have commented my code, particularly in hard-to-understand areas
[] My changes generate no new warnings or errors
[] I have added test instructions that prove my fix is effective or that my feature works
(edited)
[01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/01_Tabloid_Create_DB.sql)
<https://github.com/[NewForce-Cohort-7/tabloidfullstack-bloodyscallwags](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags)|NewForce-Cohort-7/tabloidfullstack-bloodyscallwags>NewForce-Cohort-7/tabloidfullstack-bloodyscallwags | Added by [GitHub](https://newforce-wv.slack.com/services/B02GRKT87JN)
[02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/02_Tabloid_Seed_Data.sql)
<https://github.com/[NewForce-Cohort-7/tabloidfullstack-bloodyscallwags](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags)|NewForce-Cohort-7/tabloidfullstack-bloodyscallwags>NewForce-Cohort-7/tabloidfullstack-bloodyscallwags | Added by [GitHub](https://newforce-wv.slack.com/services/B02GRKT87JN)
